docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -50,13 +48,11 @@ when the app discovery feature generates the `KeptnApp` resource:
 
 - **apiVersion** -- API version being used.
 - **kind** -- Resource type.
-   Must be set to `KeptnApp`
+   Must be set to `KeptnApp`.
 
-- **metadata**
-  - **name** -- Unique name of this application.
-    Names must comply with the
-    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
-    specification.
+- **metadata** -- Information about the resources.
+  - **name** -- This field is used to specify the unique name of the application or resource being defined. The name must follow the guidelines outlined in the [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) specification.
+  - **namespace** -- Isolates resources in Kubernetes, aiding organization and preventing naming conflicts across clusters, enhancing multi-tenant and collaborative cluster management. 
```

Comment:
This kind of defines what a namespace is but it doesn't define what information goes into this field for this CRD and what the implications are.  You might get some help from https://lifecycle.keptn.sh/docs/install/k8s/#how-many-namespaces.